modules/example/src/main/scala/zio/telemetry/example/backend/Server.scala

```diff
@@ -0,0 +1,31 @@
+package zio.telemetry.example.backend
+
+import cats.effect.ExitCode
+import org.http4s.server.Router
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.syntax.kleisli._
+import zio._
+import zio.clock.Clock
+import zio.interop.catz._
+import zio.telemetry.example.JaegerTracer._
+import zio.telemetry.example.backend.config.Configuration
+import zio.telemetry.example.backend.http.StatusService
+
+object Server extends CatsApp {
```

Comment:
You can flatten the package hierarchy. It's the single project, move it out of subpackage and name accordingly (i.e. `Backend`).